### PR TITLE
Eventbrite API: Webhooks

### PR DESCRIPTION
**Pull request by jamesbiggs**
## About these changes

This PR is for the integration of the Eventbrite Webhook, for our integration on the What's On project.

## How to check these changes

You can either CURL the local URL or just open the URL in your browser to see a failed message.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: etna/whatson/views.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+
```
Comment (Firdaussi) [inline etna/whatson/views.py:4]: Too much whitespace?

File: etna/whatson/views.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@csrf_exempt
+def eventbrite_webhook_view(request):
+    if request.method == "POST":
+        # Parse the JSON data from the request
+        # json_data = json.loads(request.body)
+
```
Comment (Firdaussi) [inline etna/whatson/views.py:10]: At this stage there are alot of unknowns, like what we receive, how much control we have over the granularity of the webhooks and whether we send an acknowledgement to EB. As a starting point, though, this looks fine.

File: etna/whatson/views.py (hunk 1 above)
Comment (jamesbiggs) [reply etna/whatson/views.py:4]: Whitespace is enforced by our formatters - they leave a double line after our imports 😃

**Review by Firdaussi — APPROVED** [review]
All looks sensible to me.

**Pull request merged**